aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -83,6 +92,11 @@ module aptos_framework::aptos_governance {
         votes: Table<RecordKey, bool>
     }
 
+    /// Records to track the voting power usage of each stake pool on each proposal.
+    struct VotingRecordsV2 has key {
+        votes: Table<RecordKey, u64>
```

Comment:
Let's also use SmartTable here instead

Reply:
@rahxephon89 I got these error in specs:

>   error: expression construct not supported in specifications
>       ┌─ /Users/xinding/src/public/aptos-core/aptos-move/framework/aptos-framework/sources/aptos_governance.spec.move:620:13
>       │
>   620 │             *smart_table::borrow(voting_records_v2.votes, record_key)
>       │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 
>   error: calling impure function `smart_table::contains` is not allowed
>       ┌─ /Users/xinding/src/public/aptos-core/aptos-move/framework/aptos-framework/sources/aptos_governance.spec.move:619:37
>       │
>   619 │         let used_voting_power = if (smart_table::contains(voting_records_v2.votes, record_key)) {
>       │                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>       │
>       = impure function `smart_table::contains(&smart_table::SmartTable<#0, #1>, #0): bool`

It seems that I need to write lots of `spec_` functions for smart_table and its dependencies. Any easy way I can do that?

Reply:
Unfortunately we do not have intrinsic implementations of smart_table APIs encoded in the prover right now. We will work on them later.

Reply:
Changed to `SmartTable`! Thanks for Teng's help on specs.